src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -58,6 +58,20 @@
     </Environment>
   </Provider>
 
+  <Provider Name="StackExchange" Documentation="https://api.stackexchange.com/docs/authentication">
+    <Environment Issuer="https://api.stackexchange.com/">
+      <Configuration AuthorizationEndpoint="https://stackoverflow.com/oauth"
+                     TokenEndpoint="https://stackoverflow.com/oauth/access_token/json"
```

Comment:
@martincostello StackExchange supports returning a JSON token response if you use `https://stackoverflow.com/oauth/access_token/json` instead of `https://stackoverflow.com/oauth/access_token/json`. Should we update the aspnet-contrib provider to use it and avoid having to deal with formurl-encoded responses?

Reply:
Cool - I'll take a look at that and the other comment to see if we can make the change as part of the v7 release.